test/percy.json

```diff
@@ -13,6 +13,13 @@
   ],
   "localhostUrl": "https://localhost:8080/",
   "supportedLanguages": [],
-  "fullTestCoverageLanguages": ["en_US"],
+  "fullTestCoverageLanguages": [
+    "en_US",
+    "es_ES",
+    "de_DE",
+    "fr_FR",
+    "it_IT",
+    "pt_PT"
```

Comment:
Just want to confirm that this is not slowing down the build. Is it running in parallel?

Reply:
They won't run in parallel, since the way the tests are expected to run are they iterate through each language. So, it was taking apprx 5mins originally and now takes approx 25mins. 

There is some documentation around parallel execution for Percy here : https://docs.percy.io/docs/parallel-test-suites.

I am happy to create a separate percy profile to run these scenarios on an on-demand or overnight/weekly basis. We can also leave this in draft until we have a working parallel solution working.

Reply:
I think it would be good to run this overnight against development branch

Reply:
Agree about this being run overnight instead if it's going to have that big of an impact on the build time.

Reply:
Sure, makes sense. Let me figure out how best to do this, then I can at least update the PR. I think the best solution might be to leave what we currently have in the 'as-is' Percy setup, which seems to be working quite well and create a new config.